chart/templates/_helpers.tpl

```diff
@@ -1163,19 +1170,23 @@ Add environment variables to configure Redis values
 */}}
 {{- define "carto.redis.existingsecret.key" -}}
 {{- if .Values.internalRedis.enabled -}}
+  {{- if .Values.cartoSecrets.redisPassword.existingSecret.name -}}
+    {{- print .Values.cartoSecrets.redisPassword.existingSecret.key -}}
+  {{- else -}}
     {{- print "redis-password" -}}
+  {{- end }}
```

Comment:
Are you sure that it shouldn't be? Looks strange but I'm not an expert on golang templates
```suggestion
  {{- end -}}
```
(Same happens some lines below, multiple times)